### PR TITLE
Adding jzaleta.dino.icu

### PR DESCRIPTION
**Pull request by jzaleta**
`jzaleta.dino.icu` → My personal twitter profile (_while I finish building my website_) https://x.com/_jzaleta

### CODE REVIEW

File: dino.icu.yaml [hunk 1]
```diff
@@ -165,6 +165,11 @@ javier: # by https://github.com/J-cordz - my personal hack club scrapbook
    ttl: 600
    type: CNAME
    value: cname.vercel-dns.com.
+   
+jzaleta: # Javi's twitter redirect
+   ttl: 600
+   type: CNAME
+   value: https://x.com/_jzaleta.
```
Comment (Muirrum) [inline dino.icu.yaml:172]: Hey! DNS records can't have paths or protocols in them.

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: dino.icu.yaml (hunk 1 above)
Comment (jzaleta) [reply dino.icu.yaml]: Oops I totally missed that, already fixed!

Comment (Muirrum) [reply dino.icu.yaml]: Hey, you still can't have paths in DNS records. I'd recommend figuring out how to redirect to your twitter profile. There should be a way to do it from whatever you're using to build your site.

Comment (jzaleta) [reply dino.icu.yaml]: I figured out that the best workaround for this was to setup the dns record with a vercel site that makes the redirect. Hope it works now.

Comment (jzaleta) [reply dino.icu.yaml]: > I figured out that the best workaround for this was to setup the dns record with a vercel site that makes the redirect. Hope it works now.

Vercel prefers setting up an A record over `cname.vercel-dns.com.` so did the change.

**Review by alx-alexpark — APPROVED** [review]
LGTM!

**Pull request merged**